src/FuseProcess.ts

```diff
@@ -22,6 +23,40 @@ export class FuseProcess {
         return this;
     }
 
+		require(close?: ((exports: any) => void)|(() => void)) {
+			function getMainExport(mdl) {
+				return mdl && mdl.FuseBox && mdl.FuseBox.mainFile ?
+						mdl.FuseBox.import(mdl.FuseBox.mainFile) :
```

Comment:
Indeed, I just understand your question : the exports are supposed to be exposed in "mdl" - that's because of a bug I think, I'll try to make a repository to expose the bug before we pull this